.github/actions/generate-builder/action.yml

```diff
@@ -77,6 +77,7 @@ runs:
 
     - name: Set up Go environment
       uses: actions/setup-go@93397bea11091df50f3d7e59dc26a7711a8bcfbe # v4.1.0
+      if: $${{ inputs.compile-builder }}
```

Comment:
afaics it was being passed as a boolean re required arg vs string.

Reply:
All inputs in GitHub Actions are basically strings so I think we'll need to do `compile-builder == 'true'` otherwise it's a test whether the string is empty or not.

https://stackoverflow.com/questions/76292948/github-action-boolean-input-with-default-value